sklearn/neighbors/dist_metrics.pyx

```diff
@@ -135,6 +138,7 @@ cdef class DistanceMetric:
     "wminkowski"    WMinkowskiDistance    p, w      ``sum(w * |x - y|^p)^(1/p)``
     "seuclidean"    SEuclideanDistance    V         ``sqrt(sum((x - y)^2 / V))``
     "mahalanobis"   MahalanobisDistance   V or VI   ``sqrt((x - y)' V^-1 (x - y))``
+    "angular"       ArccosDistance        -         ``arccos(dot(x, y) / (|x| * |y|)) / PI``
```

Comment:
This is not the same expression as what is calculated [below](https://github.com/scikit-learn/scikit-learn/pull/7829/files#diff-6cbe0a2b5d3195466380a09badfacf18R1069) (and is not a metric: can be negative).